tools/perf/perf_run.py

```diff
@@ -328,16 +382,29 @@ def run_hf_inductor(model, input_tensors, params, precision, batch_size):
     compile_time_s = end_compile - start_compile
     iters = params.get("iterations", 20)
 
-    record_llm_perf(
-        model,
-        "Inductor",
-        input_tensors,
-        precision,
-        osl,
-        batch_size,
-        iters,
-        compile_time_s,
-    )
+    if params.get("enable_cuda_graph", False):
+        with torchtrt.runtime.enable_cudagraphs(model) as cudagraphs_module:
+            record_llm_perf(
+                cudagraphs_module,
+                "Inductor",
+                input_tensors,
+                precision,
+                osl,
+                batch_size,
+                iters,
+                compile_time_s,
+            )
+    else:
+        record_llm_perf(
+            model,
+            "Inductor",
+            input_tensors,
+            precision,
+            osl,
+            batch_size,
+            iters,
+            compile_time_s,
+        )
```

Comment:
This option doesn't apply to inductor, it is only for Torch-TRT (dynamo/torch.compile backend)